File: Introduction/Java Implementation/GCD.java
//Program to find the gcd of two numbers
import java.util.Scanner;

class GCD {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the two numbers: ");
        int m = sc.nextInt(), n = sc.nextInt();
        System.out.println("GCD of " + m + " and " + n + " is " + findGCD(m, n));
    }

    private static int findGCD(int m, int n){
        while(n != 0){
            int temp = m % n;
            m = n;
            n = temp;
        }
        return m;
    }
}

File: Unit - 1/Introduction/C implementation/gcd.c
//Program to find the gcd of two given numbers
#include<stdio.h>

int gcd(int m, int n)
{
    while(n != 0)
    {
        int temp = m % n;
        m = n;
        n = temp;
    }
    return m;
}

int main()
{
    int m, n;
    printf("Enter the two numbers: ");
    scanf("%d %d", &m, &n);
    printf("GCD of %d and %d is %d\n", m, n, gcd(m, n));
    return 0;
}